src/python/bcc/__init__.py

```diff
@@ -388,7 +388,6 @@ def attach_kprobe(self, event="", fn_name="", event_re="",
         return self
 
     def detach_kprobe(self, event):
-        assert isinstance(event, str), "event must be a string"
```

Comment:
Not from anything internal, as far as I could tell. It was primarily there to make sure callers passed in strings.